examples/with-linaria/package.json

```diff
@@ -7,9 +7,10 @@
     "start": "next start"
   },
   "dependencies": {
-    "@zeit/next-css": "^1.0.1",
-    "linaria": "2.0.0-alpha.5",
+    "@babel/core": "^7.13.10",
```

Comment:
Why is this dependency required? doesn't seem to be needed in the readme of [next-linaria](https://github.com/Mistereo/next-linaria)

Reply:
Yeah, it's not required by `next-linaria` but by `linaria` itself. https://github.com/vercel/next.js/pull/23332#discussion_r600013278

Reply:
Can you add it as a `devDependency` 🙏